AsyncPoco.Tests/DatabaseTests.cs

```diff
@@ -589,14 +589,14 @@ void AssertPocos(deco a, deco b)
 		{
 			var id = await InsertRecordsAsync(1);
 			var a2 = await db.SingleOrDefaultAsync<deco>("WHERE id=@0", id);
-			Assert.AreEqual(a2.date_created.Kind, DateTimeKind.Utc);
-			Assert.AreEqual(a2.date_edited.Value.Kind, DateTimeKind.Utc);
+			Assert.AreEqual(DateTimeKind.Utc, a2.date_created.Kind);
+			Assert.AreEqual(DateTimeKind.Utc, a2.date_edited.Value.Kind);
```

Comment:
NUnit was written to have the "Expected Value" as first argument, then "Actual Value" as second argument. PetaPocoTests.cs was written with the reverse convention. I went ahead and swapped these two to follow the new NUnit convention, but then I realized the task was going to be time-consuming, so I just stopped after changing these two lines. :)